build_run_docker.sh

```diff
@@ -746,7 +724,12 @@ else
     exit 1
 fi
 
+# Make directories accessible by group members. This is used as a workaround
+# since the local (host) user likely has a different UID from the user in the
+# docker container.
+chmod g=u -R "$AUTOGENERATED_FILES_DIR"
```

Comment:
This returns `chmod: -R: No such file or directory` on Mac

Reply:
```bash
chmod --help           
chmod: illegal option -- -
usage:  chmod [-fhv] [-R [-H | -L | -P]] [-a | +a | =a  [i][# [ n]]] mode|entry file ...
        chmod [-fhv] [-R [-H | -L | -P]] [-E | -C | -N | -i | -I] file ...
```

Reply:
Fixed.